### PR TITLE
Calo time update

### PR DESCRIPTION
**Pull request by bechenard**
New calorimeter hit extraction to include correlations in the signal time measurement and a few additional calo inspector modules

### CODE REVIEW

File: CaloDiag/fcl/caloInspect.fcl [hunk 1]
```diff
@@ -84,4 +81,4 @@ physics: {
 services.SeedService.baseSeed         :  99
 services.SeedService.maxUniqueEngines :  20
 
-physics.producers.CaloHitTruthMatch.caloShowerSimCollection  : "compressDigiMCs"
```
Comment (brownd1978) [inline CaloDiag/fcl/caloInspect.fcl]: AFAIK There is no CaloDigiMC object; what does CaloDigiTruthMatch represent?

**Review by brownd1978 — COMMENTED** [review]
My only question is about the names.

File: CaloDiag/fcl/caloInspect.fcl (hunk 1 above)
Comment (bechenard) [reply CaloDiag/fcl/caloInspect.fcl]: This refers to my inability to completely eradicate the CaloDigiTruthMatch name at the first try. This should be changed now.

**Review by brownd1978 — APPROVED** [review]
Looks good.

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Review by sdifalco — APPROVED** [review]

**Review by sdifalco — APPROVED** [review]

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Pull request merged**